pkg/reconcilers/deploymentconfig.go

```diff
@@ -196,8 +196,8 @@ func DeploymentConfigPodTemplateLabelsMutator(desired, existing *appsv1.Deployme
 	return updated, nil
 }
 
-// DeploymentConfigRemoveDuplicateEnvVarMutator ensures pod env vars not duplicated
-func DeploymentConfigRemoveDuplicateEnvVarMutator(desired, existing *appsv1.DeploymentConfig) (bool, error) {
+// DeploymentConfigRemoveDuplicateEnvVarMutator ensures pod env vars are not duplicated
+func DeploymentConfigRemoveDuplicateEnvVarMutator(_, existing *appsv1.DeploymentConfig) (bool, error) {
```

Comment:
nit:  why two args? This method cleans the env var list of the deploymentconfig passed as arg.

Reply:
Not sure I understood the question correctly but if you are talking about the method signature, I think it needs to be this signature to be a `DCMutateFn`. It's just that `desired` is unused so am just setting it to `_` to show that  🤔 

https://github.com/3scale/3scale-operator/blob/70ee1aa9cfbd096a4ad9b8639f23689c79b766cf/pkg/reconcilers/deploymentconfig.go#L17

Reply:
Right, I got confused with `helper.RemoveDuplicateEnvVars`.

Then the `TestDeploymentConfigRemoveDuplicateEnvVarMutator` test can be simplified a little bit, desired can be nil and does not need to be in the `cases` struct. But I leave this up to you, as it is a nitpick.

Reply:
Ah i see, good catch ! I'll push a commit to simplify the unit tests 👍